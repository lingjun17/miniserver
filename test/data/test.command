New 1 buy 2@100
New 2 buy 2@101
New 3 buy 2@102
New 4 buy 2@102
New 5 buy 2@103
New 6 buy 2@104
print book

New 7 sell 1@120
New 8 sell 2@120
New 9 sell 2@130
print book

New 10 sell 5@102
print book

New 11 sell 5@102
print book

Cancel 999 sell 130
Cancel 9 sell 130
print book

Replace 999 sell 3@130
Replace 2 buy 5@101
print book
