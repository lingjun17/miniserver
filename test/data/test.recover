1|0|0|100.00|0.00|0|2
2|0|0|101.00|0.00|0|2
3|0|0|102.00|0.00|0|2
4|0|0|102.00|0.00|0|2
5|0|0|103.00|0.00|0|2
6|0|0|104.00|0.00|0|2
7|1|0|120.00|0.00|0|1
8|1|0|120.00|0.00|0|2
9|1|0|130.00|0.00|0|2
6|0|2|104.00|104.00|2|0
10|1|1|102.00|104.00|2|3
5|0|2|103.00|103.00|2|0
10|1|1|102.00|103.00|2|1
3|0|1|102.00|102.00|1|1
10|1|2|102.00|102.00|1|0
3|0|2|102.00|102.00|1|0
11|1|1|102.00|102.00|1|4
4|0|2|102.00|102.00|2|0
11|1|1|102.00|102.00|2|2
11|1|1|102.00|102.00|0|2
9|1|3|130.00|102.00|0|2
2|0|3|101.00|102.00|0|2
2|0|0|101.00|102.00|0|5
