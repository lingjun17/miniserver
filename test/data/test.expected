ask:
2@102.00
1@120.00 2@120.00
bid:
5@101.00
2@100.00
